hydra-tui/src/Hydra/TUI.hs

```diff
@@ -625,7 +625,7 @@ draw Client{sk} CardanoClient{networkId} s =
   drawShow = txt . (" - " <>) . show
 
 renderTime :: (FormatTime t) => t -> String
-renderTime r = formatTime defaultTimeLocale "%Dd %Hh %Mm %Ss" r
+renderTime r = formatTime defaultTimeLocale "%dd %Hh %Mm %Ss" r
```

Comment:
Is `formatTime` shortcut for non-total days different than for non-total hours & minutes?

Reply:
According to [formatTime docs](https://hackage.haskell.org/package/time-1.13/docs/Data-Time-Format.html#v:formatTime)
uppercase and lowercase have different meanings:
```
%d: total whole days    [vs] %D: whole days of week
%h: total whole hours   [vs] %H: whole hours of day
%m: total whole minutes [vs] %M: whole minutes of hour
%s: total whole seconds [vs] %S: whole seconds of minute
```